.types/lodash-find.js

```diff
@@ -0,0 +1,3 @@
+declare module "lodash.find" {
+  declare var exports: any;
+};
```

Comment:
let's try to use the default export declaration like yay:

```js
declare module "lodash.find" {
  declare export default find(list: any[]);
}
```

but w/e the correct version should be